client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -339,6 +369,7 @@ export const SelectiveExtensionsBundle = ( {
 									)
 								);
 							} }
+							disabled={ installedExtensions.length !== 0 }
```

Comment:
It's a little confusing to me to have the `main` checkbox disabled when only one of the nested checkboxes is disabled. Logically this does make sense, but this is especially confusing when the dropdown is collapsed and the user might feel forced to install all the free extensions (although they only have one already installed):
<img width="699" alt="Screen Shot 2022-03-31 at 9 57 51 AM" src="https://user-images.githubusercontent.com/2240960/161060483-3d16f689-cc62-4ebb-993e-7d8a1464bb48.png">

Reply:
Sounds good to me! I rebased this branch. The changes related to the checkboxes are in [this commit](https://github.com/woocommerce/woocommerce-admin/pull/8514/commits/4cd824f89e03c8102d6ed78380c980a4cf7fbaee).